server/app/src/main/java/com/ams/app/dto/Studentdashrequestdto.java

```diff
@@ -0,0 +1,9 @@
+package com.ams.app.dto;
+
+import lombok.Data;
+
+@Data
+public class Studentdashrequestdto {
+  private String date;
+  private int sid;
```

Comment:
What is sid? Rename it to a meaningful name

Reply:
Changed to studentId